bot/seasons/evergreen/bookmark.py

```diff
@@ -17,17 +20,27 @@ def __init__(self, bot: commands.Bot):
 
     @commands.command(name="bookmark", aliases=("bm", "pin"))
     async def bookmark(
-        self,
-        ctx: commands.Context,
-        target_message: discord.Message,
-        *,
-        title: str = "Bookmark"
+            self,
+            ctx: commands.Context,
+            target: Union[discord.Message, discord.User],
+            *,
+            title: str = "Bookmark"
     ) -> None:
-        """Send the author a link to `target_message` via DMs."""
+        """
+        Send the author a link to `target_message` via DMs.
+
+        A user mention can also be passed to get the user's last message in channel where the command is invoked.
+        """
+        if type(target) == discord.User:
+            async for message in ctx.channel.history(limit=20):  # Takes last 2 messages from the channel.
+                if message.author == target:
+                    target = message
+                    break
```

Comment:
There should be another special check here, otherwise trying to bookmarking your last message will result in the command message being bookmarked, like this:

![image](https://user-images.githubusercontent.com/19921765/78245521-ce0be000-7511-11ea-9c15-0409e0254b5b.png)
![image](https://user-images.githubusercontent.com/19921765/78245530-d237fd80-7511-11ea-951a-553740d91d3c.png)

In practice user may want to bookmark their "other" latest message instead.